static/app/views/profiling/content.tsx

```diff
@@ -213,12 +213,16 @@ function ProfilingContent({location}: ProfilingContentProps) {
                     'profiling-global-suspect-functions'
                   ) ? (
                     <Fragment>
-                      <ProfilesChartWidget
-                        chartHeight={150}
-                        referrer="api.profiling.landing-chart"
-                        userQuery={query}
-                        selection={selection}
-                      />
+                      {organization.features.includes(
+                        'continuous-profiling-ui'
+                      ) ? null : (
```

Comment:
We can add a message here for people who are looking for it later

Reply:
Yeah, I'm going to add a banner separately and likely add this to the docs pages